src/main/java/com/epam/ta/reportportal/core/analyzer/auto/impl/AnalyzerServiceImpl.java

```diff
@@ -134,15 +141,31 @@ private void analyzeItemsPartition(Launch launch, List<Long> testItemIds,
     Optional<IndexLaunch> rqLaunch = launchPreparerService.prepare(launch, toAnalyze,
         analyzerConfig);
     rqLaunch.ifPresent(rq -> {
+      var analyticsMetadata = gatherAnalyzerStatistics(rq);
       previousLaunchId.ifPresent(rq::setPreviousLaunchId);
       Map<String, List<AnalyzedItemRs>> analyzedMap = analyzerServicesClient.analyze(rq);
+
+
       if (!MapUtils.isEmpty(analyzedMap)) {
+        analyticsMetadata.put("autoAnalyzed", analyzedMap.size());
```

Comment:
let's use a different name to avoid misunderstanding in who actually started analyser. 
'analyzed'

Reply:
renamed to 'analyzed'